Flaky test_replicated_fetches_bandwidth/test.py::test_limited_fetches_for_server

[CI Report](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=80285&sha=ac636f8bebbc5f1d50b78ab6747cd9d5a75aa7cf&name_0=PR&name_1=Integration%20tests%20%28release%2C%204%2F4%29)

```
            # approximate border. Without limit we will have more than 100 MB/s for very slow builds.
>           assert median_speed <= 15, (
                "We exceeded max fetch speed for more than 15MB/s. Must be around 5 (+- 10), got "
                + str(median_speed)
            )
E           AssertionError: We exceeded max fetch speed for more than 15MB/s. Must be around 5 (+- 10), got 15.067323223866879
E           assert 15.067323223866879 <= 15
```

--- a/tests/integration/test_replicated_fetches_bandwidth/test.py
+++ b/tests/integration/test_replicated_fetches_bandwidth/test.py
@@ -64,10 +64,12 @@ def test_limited_fetch_single_table(start_cluster):
             n2_fetch_speed.append(n2_in)
             time.sleep(0.5)
 
-        median_speed = statistics.median(n2_fetch_speed)
+        # When we start fetches we have small burst because of implementation throttling
+        # Make sense to skip first measure
+        median_speed = statistics.median(n2_fetch_speed[1:])
         # approximate border. Without limit we will have more than 100 MB/s for very slow builds.
-        assert median_speed <= 15, (
-            "We exceeded max fetch speed for more than 10MB/s. Must be around 10 (+- 5), got "
+        assert median_speed <= 20, (
+            "We exceeded max fetch speed for more than 10MB/s. Must be around 10 (+- 10), got "
             + str(median_speed)
         )
 
@@ -106,7 +108,9 @@ def test_limited_send_single_table(start_cluster):
             n1_sends_speed.append(n1_out)
             time.sleep(0.5)
 
-        median_speed = statistics.median(n1_sends_speed)
+        # When we start fetches we have small burst because of implementation throttling
+        # Make sense to skip first measure
+        median_speed = statistics.median(n1_sends_speed[1:])
         # approximate border. Without limit we will have more than 100 MB/s for very slow builds.
         assert median_speed <= 10, (
             "We exceeded max send speed for more than 5MB/s. Must be around 5 (+- 5), got "
@@ -151,7 +155,9 @@ def test_limited_fetches_for_server(start_cluster):
             n3_fetches_speed.append(n3_in)
             time.sleep(0.5)
 
-        median_speed = statistics.median(n3_fetches_speed)
+        # When we start fetches we have small burst because of implementation throttling
+        # Make sense to skip first measure
+        median_speed = statistics.median(n3_fetches_speed[1:])
         # approximate border. Without limit we will have more than 100 MB/s for very slow builds.
         assert median_speed <= 15, (
             "We exceeded max fetch speed for more than 15MB/s. Must be around 5 (+- 10), got "
@@ -197,10 +203,12 @@ def test_limited_sends_for_server(start_cluster):
             n3_sends_speed.append(n3_out)
             time.sleep(0.5)
 
-        median_speed = statistics.median(n3_sends_speed)
+        # When we start fetches we have small burst because of implementation throttling
+        # Make sense to skip first measure
+        median_speed = statistics.median(n3_sends_speed[1:])
         # approximate border. Without limit we will have more than 100 MB/s for very slow builds.
         assert median_speed <= 20, (
-            "We exceeded max send speed for more than 20MB/s. Must be around 5 (+- 10), got "
+            "We exceeded max send speed for more than 20MB/s. Must be around 5 (+- 15), got "
             + str(median_speed)
         )
 

